astropy/coordinates/tests/test_matching.py

```diff
@@ -297,3 +297,14 @@ def test_match_catalog_empty():
     with pytest.raises(ValueError) as excinfo:
         sc1.match_to_catalog_3d(cat0)
     assert 'catalog' in str(excinfo.value)
+
+@pytest.mark.skipif(str('not HAS_SCIPY'))
```

Comment:
```
@pytest.mark.skipif(not HAS_SCIPY or OLDER_SCIPY, reason="needs scipy>=???")
```

Reply:
Yea, not sure why this was a string - I just took the commits from @eteq's branch